### DISCUSSION TITLE
Game engine with reanimated 2 possible?

### DISCUSSION THREAD

**Discussion by marcel-happyfloat** [software-mansion/react-native-reanimated#1723] [Q&A]
Hello,

Is it possible to execute the worklets on every frame (e.g. requestAnimationFrame) and calculate some physics or bounding-boxes?

I want to archive a similar effect such as in this example:
https://github.com/hankzhuo/react-native-bouncing-ball

But I don't think calling the bridge or setState in every frame is the right approach. Can reanimated 2 work for such a use case, how?

I'm completely new to "reanimated" in general and I only find examples related to interactions or basic A->B transitions. So my question is if it is possible to create a "game loop" with it too? Any example? Thank you very much.

**Comment by sergeymorkovkin** [comment]
Check this: https://github.com/expo/expo/issues/11267#issuecomment-782092028
